samples/TR/W3C.css

```diff
@@ -12,13 +12,13 @@ body {
 	padding: 0 !important;
 }
 @page:left {
-	margin: 10% 10% 10% 14%!important;
+	margin: 10% 10% 10% 10% !important;
```

Comment:
Symmetrical margins are good for readability on single page view, and corner page numbers are good for spread view. I thought that corner page numbers on single page view are not very good but also not so bad.